data/crac-creation/crac-creator-csa-profiles/src/main/java/com/farao_community/farao/data/crac_creation/creator/csa_profile/crac_creator/remedial_action/CsaProfileRemedialActionsCreator.java

```diff
@@ -348,4 +372,166 @@ public static boolean isOnConstraintInstantCoherent(Instant cnecInstant, Instant
+                Optional<Integer> speedOpt = getSpeedOpt(schemeRemedialActionsPropertyBag.get(CsaProfileConstants.TIME_TO_IMPLEMENT));
+
+                String remedialActionSchemeKind = remedialActionSchemePropertyBag.get(CsaProfileConstants.RA_KIND);
+                if (!remedialActionSchemeKind.equals(CsaProfileConstants.SIPS)) {
+                    throw new FaraoImportException(ImportStatus.INCONSISTENCY_IN_DATA, CsaProfileConstants.AUTO_REMEDIAL_ACTION_MESSAGE + autoRemedialActionId + " will not be imported because Unsupported kind for remedial action schedule");
+                }
+                String remedialActionSchemeNormalArmed = remedialActionSchemePropertyBag.get(CsaProfileConstants.NORMAL_ARMED);
+                if (!Boolean.parseBoolean(remedialActionSchemeNormalArmed)) {
+                    throw new FaraoImportException(ImportStatus.INCONSISTENCY_IN_DATA, CsaProfileConstants.AUTO_REMEDIAL_ACTION_MESSAGE + autoRemedialActionId + " will not be imported because normalArmed must be set to true");
+                }
+
+                PropertyBag stage = remedialActionSchemeToStageMap.get(remedialActionSchemePropertyBag);
+                PropertyBag gridStateAlterationCollection = stageToGridStateAlterationCollectionMap.get(stage);
+
+                String collectionId = gridStateAlterationCollection.get(CsaProfileConstants.MRID);
+
+                RemedialActionType remedialActionType;
+                if (linkedTopologyActionsAuto.containsKey(collectionId)) {
+                    remedialActionType = RemedialActionType.NETWORK_ACTION;
+                } else if (linkedRotatingMachineActionsAuto.containsKey(collectionId)) {
+                    checkEachInjectionSetPointActionHasExactlyOneStaticPropertyRangeElseThrowException(autoRemedialActionId, linkedRotatingMachineActionsAuto.get(collectionId), linkedStaticPropertyRanges);
+                    remedialActionType = RemedialActionType.NETWORK_ACTION;
+                } else if (linkedShuntCompensatorModificationAuto.containsKey(collectionId)) {
+                    checkEachInjectionSetPointActionHasExactlyOneStaticPropertyRangeElseThrowException(autoRemedialActionId, linkedShuntCompensatorModificationAuto.get(collectionId), linkedStaticPropertyRanges);
+                    remedialActionType = RemedialActionType.NETWORK_ACTION;
+                } else if (linkedTapPositionActionsAuto.containsKey(collectionId)) { // StaticPropertyRanges not mandatory in case of tapPositionsActions
+                    remedialActionType = RemedialActionType.PST_RANGE_ACTION;
+                } else {
+                    throw new FaraoImportException(ImportStatus.INCONSISTENCY_IN_DATA, CsaProfileConstants.REMEDIAL_ACTION_MESSAGE + collectionId + " will not be imported because there is no topology actions, no Set point actions, nor tap position action linked to that ARA");
```

Comment:
Could you replace " will not be imported because there is no topology actions, no Set point actions, nor tap position action linked to that ARA" by " will not be imported because there is no elementary action for that ARA" please ?

And also apply the same change for non-auto remedial actions